aad/aad-b2c-with-spring-security/aad-b2c-web-application-1-simple-sample/src/main/resources/templates/home.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<body>
+
+<h1>Home Page</h1>
+<div>
+    <form style="display:inline" method="get" action="/logout">
+        <button class="btn btn-md btn-primary btn-block" type="submit">Log out</button>
+    </form>
+
+    <!-- this uri is used to redirect you to edit your profile with your profile edit user flow,
+      please replace ${your-tenant-name} with your tenant name
+             replace ${your-profile-edit-user-flow-name} with your profile edit user flow name
+             replace ${your-client-id} with your `Application ID`-->
+    <form style="display:inline" method="post" action="https://${your-tenant-name}.b2clogin.com/${your-tenant-name}.onmicrosoft.com/${your-profile-edit-user-flow-name}/oauth2/v2.0/authorize?response_type=code&client_id=${your-client-id}&scope=${your-client-id}%20openid%20profile%20offline_access&redirect_uri=http://localhost:8080/">
+        <button class="btn btn-md btn-primary btn-block" type="submit">Profile edit</button>
```

Comment:
I tried `Profile edit` in my localhost.

1. Sometimes it return `http://localhost:8080/?error=server_error&error_description=AADB2C%3a+An+exception+has+occurred.%0d%0aCorrelation+ID%3a+2f87e519-9750-42ca-a6e8-f354e4ff7655%0d%0aTimestamp%3a+2021-10-11+01%3a21%3a18Z%0d%0a` Did you know this error?
2. Sometimes it require to input username and password again. We already discussed this before, did you find the solution?

Reply:
Now the 2 cases appears like this: case 1 -> case 2 -> case 1 -> case 2 ...

Reply:
If we use `Standard` user flow, it will be all ok. For `Recommended` user flow, we need to change the profile edit uri to `https://${your-tenant-name}.b2clogin.com/${your-tenant-name}.onmicrosoft.com/oauth2/v2.0/authorize?p=${your-profile-edit-user-flow-name}&client_id=${your-client-id}&redirect_uri=http%3A%2F%2Flocalhost%3A8080&scope=openid&response_type=id_token`.

We need to input username and password again, now i haven't found a suitable method yet(`Standard` is ok).